dev/sg/dependencies/ubuntu_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestUbuntuFix(t *testing.T) {
-	if !strings.Contains(*sgSetupTests, string(OSUbuntu)) {
+	if !strings.Contains(*sgSetupTests, string(OSLinux)) {
```

Comment:
```suggestion
	if !strings.Contains(*sgSetupTests, string("ubuntu")) {
```

as @burmudar notes, this is required for our integration tests